en/CONTRIBUTING.md

```diff
@@ -26,34 +26,33 @@
 
 - code blocks
 
-## Tables
+## Notes blocks
+- A single note:
+> **Note**:
+- Multiple notes:
+> **Notes**:
+> 1. some text
+> 2. some text
+> 3. some test
 
-Can bold or italics the first colume but bot the second column. __Examples__:
+## Tables
 
-| Properties | Explanation |
-| --- | --- |
-| anisotropy | Anisotropy value |
-| minFilter | Narrowing Filter Algorithm |
+Can bold the first colume but not the second column. __Examples__:
 
 | Properties | Explanation |
 | --- | --- |
```

Comment:
The table needs to be left justified:

| :--- | :--- |

Reply:
Great idea to mention this. I will make sure I fix this in tables that are not left justified.